apps/web/lib/QueryCell.tsx

```diff
@@ -9,8 +9,16 @@ import {
 
 import { Alert } from "@calcom/ui/Alert";
 
+import { trpc } from "@lib/trpc";
+
 import Loader from "@components/Loader";
 
+import type { AppRouter } from "@server/routers/_app";
+import type { TRPCClientErrorLike } from "@trpc/client";
+import type { UseTRPCQueryOptions } from "@trpc/react";
+import type { inferProcedures } from "@trpc/react/src/createReactQueryHooks";
```

Comment:
Type error: `Module '"@trpc/react/src/createReactQueryHooks"' declares 'inferProcedures' locally, but it is not exported.ts(2459)`